pytket/extensions/braket/backends/braket.py

```diff
@@ -356,9 +356,11 @@ def __init__(
             else:
                 raise ValueError(f"Unsupported device type {aws_device_type}")
         props = self._device.properties.dict()
-        try:
-            device_info = props["action"][DeviceActionType.JAQCD]
-        except KeyError:
+        action = props["action"]
+        device_info = action.get(DeviceActionType.JAQCD)
```

Comment:
Should we put this in a try block in case this fails and one of the other steps would work?

Reply:
This can't fail; `get()` will return `None` if the key doesn't exist.